analyses/tmb-compare-tcga/TMB_d3b_code/config_files/target_cfg.targetcombos.txt

```diff
@@ -0,0 +1,6 @@
+WGS_PNOC003	/Users/kogantit/Documents/TMB/target_analysis/ccds_and_target/CCDS.bed  
```

Comment:
All files listed here need to be in the repository or downloads folder. Paths should be relative, not absolute.